instructure/canvas-lms
# Adds functionality to support shortcuts for adding all members of a g…

…roup or section to a conference.

This change adds checkboxes for groups and sections to the members list section of the dialog for creating / editing a conference. These checkboxes are "shortcuts" that allow the user to quickly add all members of a specific section or group to a conference. These shortcuts are implemented client side and the  model for conferences is unchanged.

Test Plan:
	- Make sure the conferences feature is enabled
	- In a course open the dialog to create a new conference
	- Uncheck the "Invite All" box
	- Verify the following in the member list:
		- A checkbox for each section should appear in the "Sections" partition (only if there is more than one section in the course)
		- A checkbox for each group should appear in the "Groups" partition (only if there is at least one section in the course)
		- The headers for "Sections" and "Groups" should only appear if there is atleast item in the respective partition
		- The header for "Users" should only appear if one or more of the other partitions is present
		- Checking the box for section / group also checks the boxes for each of the users in that section / group
		- Unchecking the box for section / group also unchecks the boxes for each of the users in that section / group provided that another section / group that contains that user isn't also checked
		- If a user is added to the conference via a section or group you should not be able to uncheck the checkbox for that user.
